CHANGELOG.md

```
@@ -1,3 +1,25 @@
+## 2.0.0 (27th July 2026)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)